client/package.json

```diff
@@ -128,6 +128,6 @@
     ]
   },
   "engines": {
-    "node": "^6.9.0"
+    "node": "^8.4.0"
```

Comment:
Scope is being installed as a dependency, `service-ui` is not (at least that was my thinking at the time). `service-ui` has the engines key added back in now.